build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/DockerBase.java

```diff
@@ -29,7 +29,9 @@ public enum DockerBase {
     // spotless:on
     // Based on WOLFI above, with more extras. We don't set a base image because
     // we programmatically extend from the wolfi image.
-    CLOUD_ESS(null, "-cloud-ess", "apk");
+    CLOUD_ESS(null, "-cloud-ess", "apk"),
+    // Based on WOLFI above, we programmatically extend from the wolfi image.
+    FIPS(null, "-fips", "apk");
```

Comment:
we need to base the image from: chainguard-base-fips

while the majority of the FIPS compliant for ES is from the JDK/app level, the container itself must also be FIPS compliant.

Reply:
we transitively base this on `chainguard-base` as cloud-ess is based on our es-wolfi image and this is based on `chainguard-base`. @jakelandis you say this must be `chainguard-base-fips` instead?

Reply:
I need to do more tweaking as but you can test this from the branch by running:
```
./gradlew buildAarch64FipsDockerImage

# start image directly
docker run elasticsearch-fips:aarch64

# inspect image 
docker run -it --entrypoint /bin/bash elasticsearch-fips:aarch64 
```

Reply:
@jakelandis With the latest WIP I configure a hardcoded password for the keystore (as it must be >12 characters) Strictly speaking that's fips compliant as I get it, but how is that safe enough if we bake that password into the image itself? each image would have the same keystone password basically public. 

It seems we should instead have the ability in ES to ensure we generate dynamically a keystore with fips compliant password without explicitly declaring it?

Reply:
> you say this must be chainguard-base-fips instead?

Yes, the base image, for this purpose, must also be FIPS compliant.  

> but how is that safe enough if we bake that password into the image itself? each image would have the same keystone password basically public.

This probably warrants it's own discussion, there is likely some cloud level config needed here.

Reply:
@jakelandis I changed the base image to be `chainguard-base-fips`. One side effect here is that `aarch64` is not supported anymore. You can test this with 

```
./gradlew buildFipsDockerImage

# start image directly
docker run elasticsearch-fips:x86_64

# inspect image 
docker run -it --entrypoint /bin/bash elasticsearch-fips:x86_64 
```